src/card.js

```diff
@@ -0,0 +1,142 @@
+const NO_IMAGE = "./image/no-image.png";
+const LOAD_IMAGE = "./image/load-icon.png";
+const REPORT_SIZE = 5
+
+function getCard(data) {
+    const colDiv = createElementWrapper("div", ["col"], data.id);
+    const cardDiv = createElementWrapper("div", ["card", "shadow-sm"]);
+    const cardBodyTopDiv = createElementWrapper("div", ["card-body"]);
+    const rowDiv = createElementWrapper("div", ["row"]);
+    const coinID = createElementWrapper("h5", ["col-8", "card-text"]);
+    coinID.innerText = data.symbol; //data.id
+    const switchDiv = createElementWrapper("div", ["col-4", "form-check", "form-switch"]);
+    const switchInput = createElementWrapper("input", ["form-check-input", "float-end"], colDiv.id + "_switch");
+
+    //switchInput.setAttribute("data-bs-toggle", "modal")
+    //switchInput.setAttribute("data-bs-target", `[id="staticBackdrop"]`)
+
+    switchInput.addEventListener("click", (e) => {
+        const isChecked = e.target.checked
+        const index = state.selectedCoinId.findIndex((cId) => {
+            return cId.id === data.id
+        })
+        if (isChecked) {
+            const canAdd = state.selectedCoinId.length < REPORT_SIZE
```

Comment:
💯 

isLimitedCoins 

usually boolean param is more convention to start with `is` prefix